source/postcard/src/ser/flavors.rs

```diff
@@ -250,6 +250,47 @@ where
     }
 }
 
+/// Wrapper over a [`core::iter::Extend<u8>`] that implements the flavor trait
+///
+/// This is like [`ExtendFlavor`], but it takes the writer by reference instead of
+/// by value. This allows you to remain in control of the writer.
+pub struct ExtendRefFlavor<'a, T> {
+    iter: &'a mut T,
```

Comment:
Maybe call it `bytes`? `Extend` is not a trait for iterator, it represent a collection that can accept an iterator to extend itself.
```suggestion
    bytes: &'a mut T,
```